tools/reviewer/web/src/app/dashboard/reviews-panel/review/review.component.ts

```diff
@@ -45,4 +77,87 @@ export class ReviewComponent implements OnInit {
       });
     });
   }
+
+  // Get reviewers from the Diff
+  getReviewers(): void {
+    const diffReviewers = this.diff.reviewers;
+    this.reviewers = '';
+    for (let i = 0; i < diffReviewers.length; i++) {
+      this.reviewers =
+        this.reviewers +
+        (i === diffReviewers.length - 1
+          ? diffReviewers[i]
+          : diffReviewers[i] + ', ');
+    }
+  }
+
+  // Get CCs from the Diff
+  getCCs(): void {
+    const diffCCs = this.diff.cc;
+    let cc = '';
+    for (let i = 0; i < diffCCs.length; i++) {
+      cc = cc + (i === diffCCs.length - 1 ? diffCCs[i] : diffCCs[i] + ', ');
+    }
+    this.ccs = cc;
+  }
+
+  // Get bug property from Diff
+  getBug(): void {
+    this.bug = this.diff.bug;
+  }
+
+  // Save the edited CCs to the Diff and update the Diff
+  saveCCs(): void {
+    let ccs = this.ccs.split(',');
+    ccs = ccs.map(v => v.trim());
+    this.diff.cc = ccs;
+    this.updateDiff(this.diff, 'CCs Saved');
+  }
+
+  // Save the new reviewers and update Diff
+  saveReviewers(): void {
+    let reviewers = this.reviewers.split(',');
+    reviewers = reviewers.map(v => v.trim());
+    this.diff.reviewers = reviewers;
+    this.updateDiff(this.diff, 'Reviewers Saved');
+  }
```

Comment:
There are alike methods: 
`getReviewers(), getCCs(), getBug() ...`
`saveCCs(), saveCCs() ...`
Could you create one method instead for getSomething and saveSomething?
https://angular.io/guide/styleguide#t-dry-try-to-be-dry